packages/jokul/src/components/button/styles/button-group.scss

```diff
@@ -0,0 +1,10 @@
+@use "../../../core/jkl";
+
+.jkl-button-group {
+    container: --button-group / inline-size;
```

Comment:
Min første reaksjon her var _er dette en CSS variabel?_

Mulig det er min feil fordi jeg ikke har sett så mye på syntaxen til container-queries enda men jeg tror jeg ville foretrukket å droppe `--` prefix for container-name.

Reply:
Hehe, ser absolutt hvor reaksjonen kommer fra! Dette er en [dashed-ident](https://developer.mozilla.org/en-US/docs/Web/CSS/dashed-ident), som for så vidt er det samme som brukes til å navngi custom variabler. Det er i ferd med å bli standard for å navngi egendefinerte verdier i CSS.

`container-name` støtter også "vanlig" [custom-ident](https://developer.mozilla.org/en-US/docs/Web/CSS/custom-ident), så vi kan fint kjøre uten `--` her! Tenkte bare jeg skulle være litt foran kurven 😅 For eksempel støtter `anchor()` API-et for plassering _kun_ dashed-ident, så man må nok bli vant til det før eller senere uansett.